src/components/modals/EditMembersModal.tsx

```diff
@@ -202,24 +202,22 @@ export default function EditMembersModal({ open, canModifyMembers = false, ...pr
                         </div>
                     ) : (
                         <>
-                            {members.map(({ address, permissions, accountType }) => {
-                                return accountType !== AccountType.Delegated ? (
-                                    <Item
-                                        key={address}
-                                        onMenuToggle={onMenuToggle}
-                                        address={address}
-                                        canBeDeleted={canModifyMembers}
-                                        onDeleteClick={onDeleteClick}
-                                        isCurrentAccount={isSameAddress(address, account)}
-                                        isCurrentDelegatedAccount={isSameAddress(
-                                            address,
-                                            delegatedAccount
-                                        )}
-                                        permissions={permissions}
-                                        accountType={accountType}
-                                    />
-                                ) : null
-                            })}
+                            {members.map(({ address, permissions, accountType }) => (
```

Comment:
@bearni95, `members` no longer holds delegated accounts (you can see that logic in `detect.saga`). Now we don't have to filter out anything here.